Example/Tests/ApiCallRequestDataTests.swift

```diff
@@ -29,8 +28,8 @@ class ApiCallRequestDataTests: XCTestCase {
     }
     
     override func tearDown() {
-        super.tearDown()
         server.stop()
+        super.tearDown()
```

Comment:
`super.tearDown()` always goes as last: we first clean up the current class, then proceed to the upper chain.